crates/wash-lib/src/parser/mod.rs

```diff
@@ -221,6 +223,8 @@ struct RawRustConfig {
     pub cargo_path: Option<PathBuf>,
     /// Path to cargo/rust's `target` directory. Optional, defaults to `./target`.
     pub target_path: Option<PathBuf>,
+    // Whether to build in debug mode. Defaults to false.
+    pub debug: bool,
```

Comment:
Given the test failure, 
```
thread 'rust_component' panicked at crates/wash-lib/tests/parser/main.rs:17:18:
assertion failed, expected Ok(..), got Err(missing field `debug`)
```

I think the only thing we need here is to mark this as a field that can default to `false.`
```suggestion
    // Whether to build in debug mode. Defaults to false.
    #[serde(default)]
    pub debug: bool,
```

Reply:
Hey @brooksmtownsend I solved this with an `Option<T>` since it kind of matches what we do elsewhere (I also like being able to tell the difference between not set, manually set to `false`, and manually set to `true`